eks_infrastructure/helper.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
```

Comment:
was this file pushed with +x chmod permissions?

Reply:
no. it shouldn't be though cuz we are just sourcing it and not executing it.